Remove http binding by default (#216)
sky-uk/feed: CHANGELOG.md

# v3.0.0
* Breaking change 

>   By default feed will not bind http port on 8080 if `--ingress-port=XXXX` is not provided   

Reviewer: Just to avoid the double-negative :)

```suggestion
  Feed will not bind port http 8080 by default, unless `--ingress-port=XXXX` is provided   
```
